examples/README.md

```
@@ -0,0 +1,159 @@
+go mod download
+```
+
+**Container cleanup:**
+```bash
+# Remove all demo containers
+docker ps -a --filter "label=com.ourocodus.containersession.managed-by" -q | xargs docker rm -f
+```
+
+## Architecture Overview
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)